common/src/types/token.rs

```diff
@@ -383,51 +55,10 @@ pub static TOKEN_REMAPS: OnceLock<BiMap<String, String>> = OnceLock::new();
 /// to the number of decimals the token uses (fixed-point offset)
 pub static ADDR_DECIMALS_MAP: OnceLock<HashMap<String, u8>> = OnceLock::new();
 
-lazy_static! {
-    /// The mapping of ERC-20 ticker to the expected ticker names on each Exchange.
-    static ref EXCHANGE_TICKERS: HashMap<Exchange, HashMap<String, String>> = {
-        let mut exchange_tickers = HashMap::<Exchange, HashMap<String, String>>::new();
-        for exchange in [Exchange::Binance, Exchange::Coinbase, Exchange::Kraken, Exchange::Okx] {
-            exchange_tickers.insert(exchange, HashMap::<String, String>::new());
-        }
-        for (erc20_ticker, binance_ticker, coinbase_ticker, kraken_ticker, okx_ticker) in
-            TICKER_NAMES.iter()
-        {
-            let process_ticker = move |ticker: ExchangeTicker| -> Option<&'static str> {
-                match ticker {
-                    ExchangeTicker::Same => Some(erc20_ticker),
-                    ExchangeTicker::Renamed(ticker) => Some(ticker),
-                    ExchangeTicker::Unsupported => None,
-                }
-            };
-            if let Some(binance_ticker) = process_ticker(*binance_ticker) {
-                exchange_tickers
-                    .get_mut(&Exchange::Binance)
-                    .unwrap()
-                    .insert(String::from(*erc20_ticker), String::from(binance_ticker));
-            }
-            if let Some(coinbase_ticker) = process_ticker(*coinbase_ticker) {
-                exchange_tickers
-                    .get_mut(&Exchange::Coinbase)
-                    .unwrap()
-                    .insert(String::from(*erc20_ticker), String::from(coinbase_ticker));
-            }
-            if let Some(kraken_ticker) = process_ticker(*kraken_ticker) {
-                exchange_tickers
-                    .get_mut(&Exchange::Kraken)
-                    .unwrap()
-                    .insert(String::from(*erc20_ticker), String::from(kraken_ticker));
-            }
-            if let Some(okx_ticker) = process_ticker(*okx_ticker) {
-                exchange_tickers
-                    .get_mut(&Exchange::Okx)
-                    .unwrap()
-                    .insert(String::from(*erc20_ticker), String::from(okx_ticker));
-            }
-        }
-        exchange_tickers
-    };
-}
+/// The mapping from ERC-20 ticker to the set of exchanges that list the token,
+/// along with the the ticker used to fetch the token's price from the exchange
+pub static EXCHANGE_SUPPORT_MAP: OnceLock<HashMap<String, HashMap<Exchange, String>>> =
```

Comment:
can we turn `HashMap<Exchange, String>` here into a more readable type? Like `ExchangeSupport`?